src/docfx/cli/Docfx.cs

```diff
@@ -62,6 +63,8 @@ internal static async Task<int> Run(string[] args)
                 return 1;
             }
 
+            CultureInfo.CurrentCulture = CultureInfo.CurrentUICulture = new CultureInfo("en-US", false);
```

Comment:
what does `false` mean here, is `new CultureInfo("en-US")` enough?

Reply:
You are right